client/src/components/navigation.jsx

```diff
@@ -1,69 +1,71 @@
-import React, { Component } from "react";
+import { HashLink as Link } from "react-router-hash-link";
 
-export class Navigation extends Component {
-  render() {
-    return (
-      <nav
-        id="menu"
-        className="navbar navbar-default navbar-expand-sm navbar-light fixed-top"
+function Navigation(props) {
+  let { isFixed = true } = props;
+
+  return (
+    <nav
+      id="menu"
+      className={`navbar navbar-default navbar-expand-sm navbar-light ${
+        isFixed ? "fixed-top" : ""
+      }`}
+    >
+      <Link className="navbar-brand page-scroll" to="/#page-top">

```

Comment:
Curios to see why we change this to HashLink rather than just <a> , I believe you can ues NavLink as well